.github/ISSUE_TEMPLATE/aws-access-request.yml

```diff
@@ -0,0 +1,90 @@
+
+name: WIP AWS Access Request
+description: To request access to AWS
+title: Access for [individual]
+labels: ['external-request', 'operations', 'ops-access-request']
```

Comment:
Should we add the "Awaiting Approval" tag automatically or should we keep that a manual task?

Reply:
People will probably use this to _adjust_ their permissions, which might not require approval. If this was only for initial access, I would say auto-tag it.

Reply:
good point. thanks.